src/pages/Chat/steps/AgeInput.test.tsx

```diff
@@ -8,6 +8,35 @@ import { WithStorageProps } from 'utils/StorageContainer'
 import { ChatContainer } from '../state'
 import { AgeInput } from './AgeInput'
 
+it('only allows chars in age input', () => {
```

Comment:
if you want to save some lines of code i think this could be expressed in `handles form changes`, it's just one assertion we need really right? I think this would be sufficient:
https://github.com/HedvigInsurance/web-onboarding/pull/104/files#diff-10a092b577aa0665e6f67f98a1b8a4dfR36